addons/io_scene_gltf2/__init__.py

```diff
@@ -940,6 +986,9 @@ def register():
 def unregister():
     for c in classes:
         bpy.utils.unregister_class(c)
+    for f in extension_panel_unregister_functors:
+        f()
```

Comment:
Should the `extension_panel_unregister_functors` list be cleared after this?  Or Blender always makes a fresh copy of the whole addon when re-registering?

Reply:
I can clear the list, but anyway, since the function runs in a try/catch scope, there is no problem is there is a dangling function trying to remove an already removed panel.